docs/readmes/howtos/troubleshooting/user_unable_to_attach.md

```diff
@@ -50,7 +50,7 @@ PROTOCOL_ERROR 111
 
 Suggested Solution:
 
-1. Identify which parameter the UE is sending in the Attach Request and compare with the ones Magma support. You can find which parameters magma is currently supported from the code. <https://github.com/magma/magma/blob/master/lte/gateway/c/oai/tasks/nas/emm/msg/AttachRequest.c>
+1. Identify which parameter the UE is sending in the Attach Request and compare with the ones Magma support. You can find which parameters magma is currently supported from the code. <https://github.com/magma/magma/blob/master/lte/gateway/c/oai/tasks/nas/emm/msg/AttachRequest.cpp>
```

Comment:
Hm, I don't know why but I cannot open this link?
![Screenshot from 2023-01-25 16-33-59](https://user-images.githubusercontent.com/34488763/214605815-d2a2eb9c-0155-4d93-99ad-3b016ab9b073.png)

Reply:
Same

Reply:
Should be fixed, thx.